be/src/storage/rowset_merger.cpp

```diff
@@ -260,7 +260,9 @@ class RowsetMergerImpl : public RowsetMerger {
         std::vector<ColumnId> sort_key_idxes;
         if (schema.sort_key_idxes().size() > 1) {
             sort_key_idxes = schema.sort_key_idxes();
```

Comment:
is sort_key_idxes useful for another copy?

Reply:
no, changed.